src/data/contributors/bijay.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Bijay Adhikari",
+    "skills" : ["JS", "Python","Java","Sql"],
+    "cause" : "fun",
+    "image" : "Avatar2",
```

Comment:
That avatar is girl avatar, I think you mistakenly put it in your card:)
Though good cause to code. **#fun** :+1: 
![image](https://user-images.githubusercontent.com/48255244/89080253-029b8580-d3a6-11ea-9f2a-540efaaa4288.png)

Reply:
can I change avatar?